charts/policies/allow-privilege-escalation-psp/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+# This is a Helm Chart Chart.yaml file. Used to build a Rancher Helm chart.
+---
+apiVersion: v2
```

Comment:
nit: keys should all be alphabetical here in general.

Reply:
If possible, I would like to keep it as it is, keys are separated in groups and with comments.